drivers/scheduler/k8s/specs/postgres/px-postgres-storage.yaml

```diff
@@ -1,12 +1,45 @@
-##### Portworx storage class
+##### Portworx storage classes
 kind: StorageClass
 apiVersion: storage.k8s.io/v1
 metadata:
     name: px-postgres-sc
 provisioner: kubernetes.io/portworx-volume
 parameters:
+   fs: "ext4"
+   block_size: "512b"
    repl: "3"
+   aggregation_level: "1"
+   io_profile: "db"
    priority_io: "high"
+   shared: "false"
+---
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+    name: px-postgres-sc-aggr
+provisioner: kubernetes.io/portworx-volume
+parameters:
+   fs: "ext4"
+   block_size: "512b"
+   repl: "1"
+   aggregation_level: "3"
+   io_profile: "random"
+   priority_io: "high"
+   shared: "false"
+---
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+    name: px-postgres-sc-seq
+provisioner: kubernetes.io/portworx-volume
+parameters:
+   fs: "ext4"
+   block_size: "512b"
```

Comment:
One of the storage classes needs to have `snap_schedule` and `label`.

https://docs.portworx.com/scheduler/kubernetes/dynamic-provisioning.html for reference